ansible_builder/main.py

```diff
@@ -325,6 +325,8 @@ def __init__(self, definition,
             ),
             "",
             "FROM $ANSIBLE_RUNNER_IMAGE as galaxy",
+            "",
+            "USER root",
```

Comment:
have we thought of just using sudo, for our install commands?  Then, if we want we can remove the sudoers entry for the user or explain how someone can.

Reply:
We can try it, we would need to document that all prepend / append steps will need to do the same. This PR should also need some changes to make those continue working.